.github/workflows/sync.yml

```diff
@@ -0,0 +1,18 @@
+name: Sync Files
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@master
+      - name: Run GitHub File Sync
+        uses: BetaHuhn/repo-file-sync-action@v1
+        with:
+          GH_PAT: ${{ secrets.GH_PAT }}
+
+#In order for the Action to access your repositories you have to specify a Personal Access token as the value for GH_PAT (GITHUB_TOKEN will not work). [The PAT needs the full repo scope . ](https://github.com/BetaHuhn/repo-file-sync-action/discussions/31#discussioncomment-674804)
```

Comment:
```suggestion
```
This comment is not necessary on the file. Please put it as a comment on the PR.